platform/view/services/storage/vault/vaultstore_test.go

```diff
@@ -32,14 +33,20 @@ const (
 )
 
 type matrixItem struct {
-	pagination driver.Pagination
-	matcher    []types.GomegaMatcher
+	pagination   driver.Pagination
+	matcher      []types.GomegaMatcher
+	sqlForward   []string
+	argsForward  []any
+	sqlBackward  []string
+	argsBackward []any
 }
 
-var r driver.Pagination = pagination.None()
 var matrix = []matrixItem{
 	{
-		pagination: r,
+		pagination:  pagination.None(),
+		sqlForward:  []string{"SELECT * FROM test", "SELECT * FROM test LIMIT $1 OFFSET $2"},
```

Comment:
Instead of having two fields, create a type that pairs sql and args:
```
type testSql struct {
sql string
args []any
}
```

Then `matrixItem` will contain following fields:
```
type matrixItem struct {
 pagination driver.Pagination
 ...
 forward: []testSql
 backward []testSql
}
```